### PR TITLE
Add and initial run of inclusive naming action

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Add inclusive naming action to workflows
- Ran the action to remove any language
- NOTE: Many cases of non-inclusive language could not be dealt with in this moment. An issue has been made to address in the future.

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Check the action works properly
- Check the language changes make lexical sense

## Fixes

Fixes: https://github.com/canonical-web-and-design/web-squad/issues/4456

### CODE REVIEW

**Comment by petesfrench** [comment]
@bartaz Hey Bartek, not sure if this is the best place to include the action. It can't simply be run in the flow of the rest of the tests as it needs to be an externally held action so we can have a localized config.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -34,3 +34,17 @@ jobs:
         run: yarn test
         env:
           CI: true
+
+  inclusive-naming-check:
+  runs-on: ubuntu-latest
```
Comment (bartaz) [inline .github/workflows/test.yaml:39]: Is this nested properly? I think it should be indented one more level (together with lines below).

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -34,3 +34,17 @@ jobs:
         run: yarn test
         env:
           CI: true
+
+    name: inclusive-naming-check
```
Comment (bartaz) [inline .github/workflows/test.yaml:38]: I think you want to define it as separate job? Alongside the `build` above?

```suggestion
  inclusive-naming-check
```

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -34,3 +34,18 @@ jobs:
...
+
+  inclusive-naming-check:
+    name: Inclusive naming check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: woke
```
Comment (bartaz) [inline .github/workflows/test.yaml:46]: More descriptive step name

```suggestion
      - name: Check inclusive naming
```

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**